hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneFileSystem.java

```diff
@@ -840,8 +842,12 @@ public String pathToKey(Path path) {
     if (!path.isAbsolute()) {
       path = new Path(workingDir, path);
     }
+    String key = path.toUri().getPath();
+    if (!OzoneFSUtils.isValidName(path.toUri().getPath())) {
```

Comment:
Avoid unnecessary duplicate conversion:

```suggestion
    if (!OzoneFSUtils.isValidName(key)) {
```